[2025-02-21T14:55:15.734+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-21T14:55:07.031927+00:00 [queued]>
[2025-02-21T14:55:15.852+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-21T14:55:07.031927+00:00 [queued]>
[2025-02-21T14:55:15.854+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-02-21T14:55:15.893+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): kafka_data_stream> on 2025-02-21 14:55:07.031927+00:00
[2025-02-21T14:55:15.930+0000] {standard_task_runner.py:57} INFO - Started process 894 to run task
[2025-02-21T14:55:15.978+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'kafka_spark_dag', 'kafka_data_stream', 'manual__2025-02-21T14:55:07.031927+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/dag_kafka_spark.py', '--cfg-path', '/tmp/tmpfanuxwa9']
[2025-02-21T14:55:15.997+0000] {standard_task_runner.py:85} INFO - Job 51: Subtask kafka_data_stream
[2025-02-21T14:55:16.328+0000] {task_command.py:416} INFO - Running <TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-21T14:55:07.031927+00:00 [running]> on host 326d0a181362
[2025-02-21T14:55:16.869+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='kafka_spark_dag' AIRFLOW_CTX_TASK_ID='kafka_data_stream' AIRFLOW_CTX_EXECUTION_DATE='2025-02-21T14:55:07.031927+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-21T14:55:07.031927+00:00'
[2025-02-21T14:55:25.814+0000] {logging_mixin.py:154} INFO - Dataset downloaded successfully.
[2025-02-21T14:55:25.972+0000] {logging_mixin.py:154} INFO - Dataset extracted successfully.
[2025-02-21T14:55:25.974+0000] {logging_mixin.py:154} INFO - No CSV file found in extracted data.
[2025-02-21T14:55:25.977+0000] {logging_mixin.py:154} INFO - Dataset preparation failed.
[2025-02-21T14:55:25.978+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-21T14:55:26.006+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=kafka_spark_dag, task_id=kafka_data_stream, execution_date=20250221T145507, start_date=20250221T145515, end_date=20250221T145526
[2025-02-21T14:55:26.058+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-21T14:55:26.109+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
